staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc_test.go

```diff
@@ -527,6 +527,49 @@ func TestToken(t *testing.T) {
 				Groups: []string{"team1", "team2"},
 			},
 		},
+		{
+			name: "groups-distributed invalid client",
+			options: Options{
+				IssuerURL:     "{{.URL}}",
+				ClientID:      "my-client",
+				UsernameClaim: "username",
+				GroupsClaim:   "groups",
+				Client:        http.DefaultClient, // cause distributed claims fetching to fail with an unknown CA error
```

Comment:
any way to verify the error or verify this client gets called?

Reply:
Updated the test logic so that I could assert the returned error.